### ISSUE TITLE
All Accounts amount calculation isn't always correct.

### ISSUE THREAD

**Issue by shellajt** [Ledger-Software/Ledger#245]
Sometimes the all account's aggregated total sticks at -0.01. I haven't been able to figure out what causes it. It seems to be somehow related to deleting transactions
![image](https://cloud.githubusercontent.com/assets/11395587/24382597/2f305318-1325-11e7-9fcd-0938f097e8e3.png)

**Comment by LadyMopar** [comment]
This happens whenever anything the table updates, although I don't know why it says -0.01. I ran into this while I was working on the accounts box, and thought it only did it upon importing or adding transactions. I could fix it by updating the accounts list view upon these actions. Apparently it does this on any change to any of the information in the table. There's a method in the MainPageController class that I used to handle the two cases I noticed, but I guess it needs to be called upon every action that happens in the table... Or something else is wrong that I missed completely.
